app/components/footer/talk_to_us_component.html.erb

```diff
@@ -20,12 +20,12 @@
             <p>Monday to Friday between <strong>8:30am and 5:30pm</strong></p>
           </div>
           <div>
-            <h3 class="heading-s">Get a teacher training adviser</h3>
+            <h3 class="heading-s">Get free, one-to-one support</h3>
             <p>
-              Teacher training advisers provide free, one-to-one support wherever you are in your journey into teaching. Whether it's just an idea, or you're looking to get back into the classroom, you could get personalised help and support.
+              Advisers provide guidance wherever you are in your journey into teaching. Whether it's just an idea, or you're looking to get back into the classroom, you could get personalised help and support.
             </p>
 
-            <a href="/tta-service" class="button">
+            <a href="https://adviser-getintoteaching.education.gov.uk/" class="button">
```

Comment:
Ahh so this was actually because the /tta-service link wasn't working in the previews - this was something I was going to ask about (and forgot!)

Didn't change for any actual reason other than the links weren't working for me so happy to keep/change back as you both see fit @gemmadallmandfe @ethax-ross

Reply:
That's odd it should definitely work in the previews - I checked out another review instance and it's working 🤔 

https://review-get-into-teaching-app-2765.london.cloudapps.digital/tta-service

Reply:
Are we clear as to the benefits and downsides of using /tta-service rather than the full link @EllieNodder @ethax-ross ?

Reply:
The benefit is it will redirect to the correct version of the adviser service (so dev -> dev, test -> test, review -> dev, prod -> prod). There shouldn't be any downsides to using it (unless it isn't working of course 😄) -- shall we put this PR back into the state it was when it didn't work and then I can see what's going on?

Reply:
Yeah sure - so should all funnel links be /tta-service? (It might've just been my code spaces playing up)

Reply:
Yeah all links to the adviser service should go through `/tta-service` 👍

Reply:
Before you change anything, can we just check with Shivani as to whether we could get some tracking benefits from using the full URL (potentially with the addition of UTM parameters)?

Reply:
You can add UTM parameters to the `/tta-service` path and it will pass them along when it redirects:

https://review-get-into-teaching-app-2765.london.cloudapps.digital/tta-service?utm_source=source&utm_medium=medium

Reply:
I'm back in VS Code and when I switch it to /tta-service, it's not working in the preview. Shall I push the changes so you can take a peek?

Reply:
Ah sorry I misunderstood - I thought you meant it wasn't working in the review app; those links won't work locally (in VS Code) but we could make them go to dev or something if that would be useful?

Reply:
Ahh right okay, that makes sense.